recipes/cmaes/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cmaes" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c5529b94d6739569b635cc33649c924ce038f3519f6a2db003ce368602d40cbe
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
```

Comment:
I do not think you need `numpy` to build the package so it can likely be removed under `host`